src/common/tests/common_test_runner.c

```diff
@@ -31,9 +31,9 @@
 #include <cmocka.h>
 #include <gurt/debug.h>
 
-#include "misc_tests.h"
-#include "daos_checksum_tests.h"
-
+/** Test Suite Function Declarations */
+int daos_checksum_tests_run();
+int misc_tests_run();
```

Comment:
(style)  Bad function definition - int misc_tests_run() should probably be int misc_tests_run(void)
(style)  externs should be avoided in .c files